src/app/register-email-form/register-email-form.component.ts

```diff
@@ -99,7 +99,7 @@ export class RegisterEmailFormComponent implements OnDestroy, OnInit {
       Validators.email,
       // Regex pattern borrowed from HTML5 specs for a valid email address:
       // https://html.spec.whatwg.org/multipage/input.html#valid-e-mail-address
-      Validators.pattern('^[a-zA-Z0-9.!#$%&\'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$')
+      Validators.pattern('^[a-zA-Z0-9.!#$%&\'*+\/=?^_`{|}~-]+@(?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])\\.(?:[a-zA-Z0-9](?:\\.[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$')
```

Comment:
Why has this been modified?  The pattern here is the same one noted in the  HTML5 docs here: https://html.spec.whatwg.org/multipage/input.html#valid-e-mail-address

When I test this at https://demo.dspace.org/register, I've found that I **already can add an email with a longer TLD**.   So, I think this pattern was already correct, and just needed to be copied elsewhere (or moved into a constant to be reused elsewhere)